docs/workflow/building/libraries/cross-building.md

```diff
@@ -87,6 +87,10 @@ You can also build just managed code with:
 
 The output is at `artifacts/bin/[BuildSettings]` where `BuildSettings` looks something like `net5.0-<TargetOS>-Debug-<Architecture>`. Ex: `artifacts/bin/net5.0-Linux-Debug-x64`. For more details on the build configurations see [project-guidelines](/docs/coding-guidelines/project-guidelines.md)
 
+Note that by default ILLinker trimming is enabled and libs built above for `arm` can't be used on other arches. To build architecture-independent libs use next command:
+
+    $ ./build.sh --arch arm /p:BuildNative=false /p:ILLinkTrimAssembly=false
+
 Building corefx for Linux ARM Emulator
```

Comment:
This has a lot of outdated references to "corefx". We do not have "corefx" anymore.